src/org/joml/Rectangled.java

```diff
@@ -433,6 +433,37 @@ public Rectangled union(Rectangled other, Rectangled dest) {
         return dest;
     }
 
+    /**
+     * Adds extents for each edge of a Rectangle.
```

Comment:
Maybe the docstring could be a bit more verbose to explain what is happening here (not everyone is familiar with what _extents_ describes here exactly. Make it clearer that adding and extent of _e_ will increase the overall size by _2x e_ in the respective direction.